tests/test_RBAC.py

```diff
@@ -58,6 +56,58 @@ def test_encrypt_request_payload(consumer_wallet, publisher_wallet, monkeypatch)
         "type": "address",
         "value": publisher_wallet.address,
     }
+    assert payload["data"] == [document]
+
+    ddo = copy.deepcopy(json_dict)
+    req["data"] = ddo
+
+    validator = RBACValidator(request_name="EncryptRequest", request=req)
+    payload = validator.build_payload()
+    assert payload["data"] == ddo
+
+
+@pytest.mark.unit
+def test_wrong_encrypt_request_payload(consumer_wallet, publisher_wallet, monkeypatch):
+    monkeypatch.setenv("PRIVATE_PROVIDER", "1")
+    req = {
+        "publisherAddress": publisher_wallet.address,
+    }
+    validator = RBACValidator(request_name="EncryptRequest", request=req)
+    with pytest.raises(Exception) as err:
+        validator.build_payload()
+    assert err.value.args[0] == "Data to encrypt is empty."
```

Comment:
you can simplify testing using `match` as an argument of `raises` instead of `err.value.args`